internal/website/content/howto/docstore/open-collection.md

```diff
@@ -0,0 +1,187 @@
+---
+title: "Open a Collection"
+date: 2019-06-08T15:14:26-04:00
+draft: true
+weight: 1
+---
+
+The first step in using Docstore is connecting to your
+document store provider. Every document store provider is a little different, but the Go CDK
+lets you interact with all of them using the [`*docstore.Collection`][] type. 
+
+[`*docstore.Collection`]: https://godoc.org/gocloud.dev/internal/docstore#Collection
+
+<!--more-->
+
+## Constructors versus URL openers
+
+If you know that your program is always going to use a particular 
+provider or you need fine-grained control over the connection settings, you
+should call the constructor function in the driver package directly (like
+`mongodocstore.OpenCollection`). However, if you want to change providers based on
+configuration, you can use `docstore.OpenCollection`, making sure you ["blank
+import"][] the driver package to link it in. See the
+[documentation on URLs][] for more details. This guide will show how to use
+both forms for each storage provider.
+
+["blank import"]: https://golang.org/doc/effective_go.html#blank_import
+[documentation on URLs]: {{< ref "/concepts/urls.md" >}}
+
+## DynamoDB {#dynamodb}
+
+The
+[`dynamodocstore`](https://godoc.org/gocloud.dev/internal/docstore/dynamodocstore) 
+package supports [Amazon DynamoDB](https://aws.amazon.com/dynamodb).
+A Docstore collection corresponds to a DynamoDB table.
+
+### DynamoDB URLs {#dynamodb-url}
+
+DynamoDB URLs provide the table, partition key field and optionally the sort key
+field for the collection.
+
+{{< goexample
```

Comment:
I haven't tested, but do shortcodes still work if you split it over multiple lines?

Reply:
I'll confirm.

Reply:
Yeah, that works.